.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,61 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v2
+        with:
+          script: |
+            // Update the following two constants each year
+            const FILTER_LABEL = '2020 chapter'
+            const TRACKER_ISSUE_ID = 989
+
+            const parseTasks = text => {
+              const tasks = []
+              text.split('\n').forEach(line => {
+                if (/^(\s{2,})?([*+-]|\d+\.)\s+\[ \]\s+\S+/.test(line)) {
+                  tasks.push('☐')
```

I think an empty table cell would appear a bit cleaner and still convey "incomplete".
```suggestion
                  tasks.push('')
```